spel/scripts/amigen7-build.sh

```diff
@@ -340,12 +398,13 @@ do
     ln "${RPM}" "${ELBUILD}"/AWSpkgs/
 done
 
-echo "Executing DiskSetup.sh"
-bash -eux -o pipefail "${ELBUILD}"/DiskSetup.sh -b "${BOOTLABEL}" -v "${AMIGENVGNAME}" -d "${AMIGENBUILDDEV}" -p "${AMIGENSTORLAY}" -B "${AMIGENBOOTSIZE}" || \
+# Invoke disk-partitioner
+ComposeDiskSetupString
+bash -euxo pipefail "${ELBUILD}"/"${DISKSETUPCMD[*]}" || \
```

Comment:
Ok, using the array did the same thing, because the script and it's args are all bound up together in one var. Let's try this:

```suggestion
bash -euxo pipefail "${ELBUILD}/DiskSetup.sh" "${DISKSETUPARGS[*]}" || \
```